monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -784,6 +784,24 @@ class ReportPageComponent extends AuthComponent {
     );
   }
 
+  generateDrupalIssue(issue) {
+    return (
+      <>
+        Upgrade Drupal server to versions 8.5.11, 8.6.10, or later.
+        <CollapsibleWellComponent>
+          Drupal server at <span className="badge badge-primary">{issue.machine}</span> (<span
+          className="badge badge-info" style={{margin: '2px'}}>{issue.ip_address}</span>) is vulnerable to <span
+          className="badge badge-danger">remote command execution</span> attack.
+          <br/>
+          The attack was made possible because the server is using an old version of Drupal.
```

Comment:
Add something like `...and had a RestAPI enabled` or something like that

Reply:
6184400f514de122cb88d9b41d85758d369e9b37